pkgs/misc/vim-plugins/generated.nix

```diff
@@ -81,6 +81,17 @@ let
     };
   };
 
+  ansible-vim = buildVimPluginFrom2Nix {
```

Comment:
you should also commit your update https://github.com/NixOS/nixpkgs/blob/41e373204912ba6d0b980212fbf1657ffbd321c0/pkgs/misc/vim-plugins/vim-plugin-names